py/private/venv.tmpl.sh

```diff
@@ -1,35 +1,15 @@
 #!/usr/bin/env bash
-# NB: we don't use a path from @bazel_tools//tools/sh:toolchain_type because that's configured for the exec
-# configuration, while this script executes in the target configuration at runtime.
 
 {{BASH_RLOCATION_FN}}
+
 runfiles_export_envvars
 
 set -o errexit -o nounset -o pipefail
 
-PWD="$(pwd)"
-
-# Returns an absolute path to the given location if the path is relative, otherwise return
-# the path unchanged.
-function alocation {
-  local P=$1
-  if [[ "${P:0:1}" == "/" ]]; then
-    echo -n "${P}"
-  else
-    echo -n "${PWD}/${P}"
-  fi
-}
+{{PRELUDE}}
 
-VENV_TOOL="$(rlocation {{VENV_TOOL}})"
-VENV_ROOT="${BUILD_WORKING_DIRECTORY}"
-VIRTUAL_ENV="$(alocation "${VENV_ROOT}/{{ARG_VENV_LOCATION}}")"
+source "$(rlocation "{{VENV}}")"/bin/activate
 
-"${VENV_TOOL}" \
-    --location "${VIRTUAL_ENV}" \
-    --python "$(alocation $(rlocation {{ARG_PYTHON}}))" \
-    --pth-file "$(rlocation {{ARG_PTH_FILE}})" \
-    --pth-entry-prefix "${RUNFILES_DIR}" \
-    --collision-strategy "{{ARG_COLLISION_STRATEGY}}" \
-    --venv-name "{{ARG_VENV_NAME}}"
+{{PREEXEC}}
 
-echo "Created virtualenv in ${VIRTUAL_ENV}"
+exec "$(rlocation "{{VENV}}")"/bin/python {{INTERPRETER_FLAGS}} "$@"
```

Comment:
previously this rule was used for IDE support via `bazel run :venv` to create the .venv symlink in the source tree. we are probably breaking those people here.